app/templates/conversations/conversation.hbs

```diff
@@ -1,5 +1,6 @@
 {{conversations/conversation-thread
   conversation=conversation
+  conversationParts=conversation.sortedConversationParts
```

Comment:
This ensures `didUpdateAttrs` is ran again when conversationParts has been updated.